build/Avalonia.Desktop.props

```diff
@@ -1,6 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <ItemGroup>
-    <PackageReference Include="Avalonia.Desktop" Version="11.0.0-preview4" />
+    <PackageReference Include="Avalonia.Desktop" Version="11.0.999-cibuild0026651-beta" />
```

Comment:
Newer CI build has fixed mobile and trimming builds. Worth to target it.

Reply:
Updated